department-of-veterans-affairs/caseflow
# [Reader] Annotations in redux

Connect #1854. This does not actually implement #1854, but once this is in, we can do #1854 quickly.

Sorry this is huge. @mdbenjam and I agreed for an earlier PR (#1760) that these PRs are best done by ripping off the bandaid instead of staying in a half-redux / half-setState world.

It will be good to get this in as quickly as possible; basically any other change to annotation handling will cause merge conflicts.

## To Test
Verify:
1. All existing annotation functionality still works
1. You can edit multiple annotations at once (by clicking "edit")
1. All annotation functionality is done via redux
1. No dead code remains
